swmm_views/08_vw_swmm_outfalls.sql

```diff
@@ -36,28 +44,36 @@ UNION
 SELECT
   wn.obj_id as Name,
   coalesce(wn.bottom_level,0) as InvertElev,
-  'FREE'::varchar as Type,
-  NULL as StageData,
+--  'FREE'::varchar as Type,
+  CASE 
+    WHEN waterlevel_hydraulic IS NOT NULL THEN 'FIXED'
```

Comment:
Looks like this attribute is only available in discharge points in this model

`waterlevel_hydraulic`

It would be better to be explicit like `dp.waterlevel_hydraulic` to avoid missing joins